licenses/protonmail-bridge-EULA

```diff
@@ -0,0 +1,33 @@
+End User License Agreement ("Agreement")
```

Comment:
You need to add it to `profiles/license_groups`, into `EULA` group.

Reply:
added